tests/config/traceViewerFixtures.ts

```diff
@@ -73,6 +71,13 @@ class TraceViewerPage {
     this.displayCanvasContentSetting = page.locator('.setting').getByText('Display canvas content');
   }
 
+  stackFrames(selected?: boolean) {
+    const entry = this.page.getByRole('list', { name: 'Stack trace' }).getByRole('listitem');
```

Comment:
Ideally, we pass `{ selected }` into `getByRole`. Perhaps we are missing `aria-selected` somewhere?

Reply:
`aria-selected` is unfortunately not valid on `listitem`s: https://www.w3.org/TR/wai-aria-1.2/#aria-selected

We then throw that its not valid (we add `aria-listed` on the listitem).

Reply:
Interesting 😄 I guess it should be `aria-current` then, but we don't have an option for this in `getByRole()`.